dev-libs/boehm_gc/boehm_gc-7.6.0.recipe

```diff
@@ -10,7 +10,7 @@ COPYRIGHT="1988, 1989 Hans-J. Boehm, Alan J. Demers
 	1991-1996 Xerox Corporation.
 	1996-1999 Silicon Graphics.
 	1999-2011 Hewlett-Packard Development Company, L.P."
-LICENSE="BOEHM"
+LICENSE="Boehm"
 REVISION="2"
```

Comment:
**Question:** Should we increment `REVISION` (here, for the old recipe, `boehm_gc-7.6.0.recipe`), since we're making a cosmetic change in `LICENSE`?

(The rule is that if we are not sure we can keep `REVISION` unchanged, then we should increment it. But I don't think it would be worth incrementing it just to trigger a new build, which would be almost identical to the previous one.)

@mmlr and/or @pulkomandy, what would you recommend here?

Reply:
The resulting package would be (even visibly) different, so the revision needs to be bumped.